portfolio.js

```diff
@@ -22,3 +22,244 @@ for (let i = 0; i < navLinks.length; i += 1) {
+    featured_image: '#',
+    long_description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release",
+    technologies: ['Ruby on rails', 'css', 'javascript', 'html'],
+    link_to_Source: '#',
+    link_to_live_version: '#',
+    mobile_popup_technologies: ['Ruby on rail', 'css', 'Javascript'],
+    desktop_popup_technologies: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+  },
+
+  {
+    title: 'calculator',
+    popup_title: 'Keeping track of hundreds of components',
+    featured_image: '#',
+    long_description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release",
+    technologies: ['Ruby on rails', 'css', 'javascript', 'html'],
+    link_to_Source: '#',
+    link_to_live_version: '#',
+    mobile_popup_technologies: ['Ruby on rail', 'css', 'Javascript'],
+    desktop_popup_technologies: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+  },
+
+];
+
+const projectsContainer = document.getElementById('projects-container');
+
+function showModal() {
+  if (!!document.querySelector('.popup-wrapper') === true) {
+    document.querySelector('.popup-wrapper').remove();
+  }
+  document.querySelector('.popup-wrapper').style.display = 'block';
+}
+
+// dynamically create the project section
+
+for (let i = 0; i < projectsDetails.length; i += 1) {
+  const projectFile = document.createElement('div');
+  projectFile.classList.add('project-file');
+
+  const projectImage = document.createElement('div');
+  projectImage.classList.add('project-image');
+
+  const imagePlaceholder = document.createElement('div');
+  imagePlaceholder.classList.add('image-placeholder');
+
+  projectImage.appendChild(imagePlaceholder);
+
+  projectFile.appendChild(projectImage);
+
+  const projectText = document.createElement('div');
+  projectText.classList.add('project-text');
+
+  const projectTitle = document.createElement('p');
+  projectTitle.classList.add('project-title');
+  projectTitle.textContent = projectsDetails[i].title;
+
```

Comment:
**[OPTIONAL]**

- Hey, I know we can do pretty much with javascript on manipulating the DOM, however, let's keep in mind enforcing the use of semantic tags to better convey the message we're passing.  Above, you've used the `<p>` tag for a `<header> || <h1> - <h6>` tag, as this is semantically incorrect. The `p` tag is used for paragraphs in HTML and the `<h1 - h6>` is used for headings in HTML. Please, kindly ensure the use of the appropriate tags in your HTML elements.